lib/test-collection.js

```diff
@@ -52,7 +52,7 @@ TestCollection.prototype.add = function (test) {
 		test.title = null;
 	}
 
-	if (!test.title) {
+	if (!test.title || test.title === 'fn') {
```

Comment:
Yeah, that's the issue. I'm trying to think of a way that we can handle both falsy values and the test's function names but that also won't affect anyone else :/